app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -5,23 +5,34 @@ class Proofer
     AUTH_TOKEN_CACHE_KEY = :usps_ippaas_api_auth_token
     # Automatically refresh our auth token if it is within this many minutes of expiring
     AUTH_TOKEN_PREEMPTIVE_EXPIRY_MINUTES = 1.minute.freeze
+    # Sponsor ID to use for enhanced in-person proofing
+    USPS_EIPP_SPONSOR_ID = 36
+    # Assurance Level to use for enhanced in-person proofing
+    USPS_EIPP_ASSURANCE_LEVEL = '2.0'
 
     # Makes HTTP request to get nearby in-person proofing facilities
-    # Requires address, city, state and zip code.
+    # Requires address, city, state, zip code and vector of trust.
     # The PostOffice objects have a subset of the fields
     # returned by the API.
     # @param location [Object]
+    # @param vector_of_trust [Array<String>]
     # @return [Array<PostOffice>] Facility locations
-    def request_facilities(location)
+    def request_facilities(location:, vector_of_trust: nil)
       url = "#{root_url}/ivs-ippaas-api/IPPRest/resources/rest/getIppFacilityList"
       body = {
         sponsorID: sponsor_id,
         streetAddress: location.address,
         city: location.city,
         state: location.state,
         zipCode: location.zip_code,
-      }.to_json
+      }
+
+      if UspsInPersonProofing::EippHelper.is_eipp?(vector_of_trust)
+        body[:IPPAssuranceLevel] = USPS_EIPP_ASSURANCE_LEVEL
+        body[:sponsorID] = USPS_EIPP_SPONSOR_ID
+      end
 
+      body = body.to_json
```

Comment:
I'm curious to hear feedback on lines 28-35.  Maybe there's a better way to modify the request body we send for users going through EIPP?